test/integration/connect/envoy/run-tests.sh

```diff
@@ -877,7 +877,7 @@ function common_run_container_tcpdump {
 
     # we cant run this in circle but its only here to temporarily enable.
 
-    docker build -t envoy-tcpdump -f Dockerfile-tcpdump .
+    retry_default docker build -t envoy-tcpdump -f Dockerfile-tcpdump .
```

Comment:
I don't think this is needed as it do not run in CI, but I'm ok leaving it for consistency.

Reply:
Despite it not being used, the test still reports a failure if there are build errors: https://app.circleci.com/pipelines/github/hashicorp/consul/36619/workflows/d9c76caf-571f-4c0a-af3c-aee9f4469b7c/jobs/806598/tests#failed-test-2